### PR TITLE
Add make .git target

### PR DESCRIPTION
**Pull request by baude**
Runs gomfmt and gitvalidation

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
Why is this named .git? Special significance?

If not, can we get a more clear name? `make validate-branch` or something like that?

**Comment by baude** [comment]
ive seen it used on other projects

**Comment by mheon** [comment]
Whitespace errors in the commit according to gitvalidation

**Comment by baude** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
I'm fine with this although there seems to be no users of it, other then Humans
LGTM

File: Makefile [hunk 1]
```diff
@@ -242,3 +242,6 @@ install.tools: .install.gitvalidation .install.gometalinter .install.md2man
 	uninstall \
 	shell \
 	changelog
+
+.PHONY: .git
```
Comment (mheon) [inline Makefile]: Don't we already have a .PHONY section?

**Comment by rhatdan** [comment]
@baude could you clean this one up so we can get it off the list.  I think the only thing needed is to move you .phony section to the global section.

**Comment by mheon** [comment]
LGTM
@rh-atomic-bot r+

**Pull request closed without merging**